package.json

```diff
@@ -276,7 +276,7 @@
   "private": true,
   "dependencies": {
     "@department-of-veterans-affairs/component-library": "1.0.0",
-    "@department-of-veterans-affairs/formation": "6.13.1",
+    "@department-of-veterans-affairs/formation": "6.13.2",
```

Comment:
to pull in [this fix](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/585)